src/service/member-api-adapter.js

```diff
@@ -38,35 +40,49 @@ module.exports = function (config) {
     var urlConfig = new ConfigService(serviceOptions).get('server');
 
     var transportOptions = $.extend(true, {}, serviceOptions.transport, {
-        url: urlConfig.getAPIPath('member/local')
+        url: urlConfig.getAPIPath(apiEndpoint)
     });
-    var http = new TransportFactory(transportOptions);
+
+    var authorizationHeader = function (token) {
+        if (token) {
+            return {
+                headers: {
+                    'Authorization': 'Bearer ' + token
+                }
+            };
+        }
+        return {};
+    };
+    var http = new TransportFactory(transportOptions, authorizationHeader(serviceOptions.token));
 
     var publicAPI = {
 
-        getGroupsByUser: function (options) {
-            var httpOptions = $.extend(true, serviceOptions, options);
-            if (!httpOptions.userId) {
+        getGroupsByUser: function (params, options) {
+            options = options || {};
+            var httpOptions = $.extend(true, serviceOptions,
+                options,
+                authorizationHeader(options.token)
```

Comment:
It was because of this case that I created the helper function above, I thought it was helpful in some cases, like in the auth manager but I can create a new api adapter every time